tests/Tunnelite.Tests/TunnelTests.cs

```diff
@@ -0,0 +1,31 @@
+using Xunit;
+
+namespace Tunnelite.Tests
+{
+    public class TunnelTests
+    {
+        [Fact]
+        public void TestTunnelConnection()
+        {
+            // Arrange
+            var tunnel = new Tunnel();
+
+            // Act & Assert
+            Assert.NotNull(tunnel);
```

Comment:
This test is quite basic. Consider adding validation of the tunnel's properties and state after creation. For example, check if the default configuration is properly set.